Automattic/jetpack
# VideoPress: add Poster panel to video block sidebar

This PR introduces the new `PosterPanel` component used to handle the video poster from the block settings sidebar.
Although I consider there are more tasks to do, I think the approach is good to be merged as it, and continue iterating in follow-up tasks. 
For instance, we'd like to set the video poster but from a video frame instead of an static image.

Fixes https://github.com/Automattic/jetpack/issues/28781

## Proposed changes:

* VideoPress: add Poster panel to video block sidebar

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to Block editor
* Add/edit a VideoPress video block
* Open the block settings sidebar 
* Confirm you see the Poster panel
* Confirm you can add, remove and replace the image from there.

https://user-images.githubusercontent.com/77539/221194222-7e27351c-220c-461a-8461-b7249b928ff0.mov
